src/main/resources/en/internet.yml

```diff
@@ -84,4 +84,32 @@ en:
           - Mozilla/5.0 (iPhone; CPU iPhone OS 12_0_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Mobile/15E148 Safari/604.1
           - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/534.59.10 (KHTML, like Gecko) Version/5.1.9 Safari/534.59.10
           - Mozilla/5.0 (iPad; CPU OS 9_3_5 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13G36 Safari/601.1
-
+      bot_user_agent:
```

Comment:
We could enrich this list using the following pages.

- https://developers.whatismybrowser.com/useragents/explore/software_type_specific/crawler/
- https://developers.google.com/search/docs/crawling-indexing/overview-google-crawlers

Reply:
I'm happy to accept this PR, would you be willing to add more bot names?